source/features/linkify-user-labels.tsx

```diff
@@ -21,6 +21,8 @@ function init(signal: AbortSignal): void {
 	observe([
 		'.tooltipped[aria-label*="a member of the"]',
 		'.tooltipped[aria-label^="This user has previously committed"]',
+		'span[data-testid=comment-author-association][aria-label*="a member of the"]',
+		'span[data-testid=comment-author-association][aria-label^="This user has previously committed"]',
```

Comment:
Yes these look just about right! Just the testid value needs to be quoted too, for clarity. 

Also:

- place them above the old ones
- add a comment above the old ones like "// PRs and pre-issue redesign 2024"